2.0/m_geoipban.cpp

```diff
@@ -62,6 +75,9 @@ class ModuleGeoIPBan : public Module
 		gi = GeoIP_new(GEOIP_STANDARD);
 		if (gi == NULL)
 			throw ModuleException("Unable to initialize geoip, are you missing GeoIP.dat?");
+		fi = GeoIP_open("/usr/share/GeoIP/GeoIPv6.dat", GEOIP_STANDARD);
```

Comment:
Is there no way to detect this file location automatically like what GeoIP_new does?

Reply:
Looking at the library it seems like you can do `GeoIP_open_type(GEOIP_COUNTRY_EDITION_V6, GEOIP_STANDARD);` maybe?